src/styles/bloks/icon.scss

```diff
@@ -0,0 +1,26 @@
+.icon {
+  height: 24px;
+  width: 24px;
+  display: block;
+  background-image: url(../images/icon/Icon-Burger-menu.svg);
```

Comment:
* Add background-repeat property here
* add height and width properties to variables